sdks/java/io/jms/src/test/java/org/apache/beam/sdk/io/jms/JmsIOIT.java

```diff
@@ -156,12 +165,14 @@ public JmsIOIT(
       String brokerUrl,
       Integer brokerPort,
       String forceAsyncAcksParam,
-      Class<? extends ConnectionFactory> connectionFactoryClass) {
+      Class<? extends ConnectionFactory> connectionFactoryClass,
+      boolean useProviderFn) {
```

Comment:
Would you mind sync the order of constructor parameter with CommonJms ? that is swap connectionFactoryClass / useProviderFn

Reply:
I guess there's a chance I won't be needing this parameter anymore when I apply your suggestion about not doubling the number of tests